portfolio/src/main/webapp/journal.js

```diff
@@ -0,0 +1,9 @@
+
+function populateAnalysis() {
```

Comment:
This doesn't seem to do anything, and is broken (you don't have a doGet in you /journal servlet).  Maybe just remove this from the PR and add it later when you can support it?

Reply:
fixed function to make it work.